amethyst_rendy/src/types.rs

```diff
@@ -12,10 +12,13 @@ pub trait Backend: rendy::hal::Backend {
 macro_rules! impl_backends {
     ($($variant:ident, $feature:literal, $backend:ty;)*) => {
 
+
         impl_single_default!($([$feature, $backend]),*);
 
-        #[cfg(not(any($(feature = $feature),*)))]
-        pub type DefaultBackend = rendy::empty::Backend;
+        static_assertions::assert_cfg!(
+            any($(feature = $feature),*),
+            concat!("You must specify at least one graphical backend feature.: ", stringify!($($feature),* "See the wiki article https://github.com/amethyst/amethyst/wiki/GraphicalBackendError for more details."))
```

Comment:
Hmm, won't this print them with no whitespace / comma in between?

Reply:
Nope, I tested the output and tweaked it a bit to make sure it appeared clearly.

`jaynus@jaynus-home-desktop:~/dev/amethyst$ cargo build --example rendy --features "animation gltf"
   Compiling amethyst_rendy v0.1.0 (/home/jaynus/dev/amethyst/amethyst_rendy)
error: You must specify at least one graphical backend feature.: "metal" , "vulkan" , "empty"
"See the wiki article https://github.com/amethyst/amethyst/wiki/GraphicalBackendError for more details."
`